nvflare/app_common/executors/multi_process_executor.py

```diff
@@ -144,11 +160,12 @@ def initialize(self, fl_ctx: FLContext):
     def _initialize_multi_process(self, fl_ctx: FLContext):
 
         try:
-            self.open_ports = get_open_ports(self.num_of_processes * 3)
             client_name = fl_ctx.get_identity_name()
             job_id = fl_ctx.get_job_id()
 
+            engine = fl_ctx.get_engine()
```

Comment:
should we use `self.engine` or get engine from the `fl_ctx`?

Reply:
Actually it needs to get engine from `fl_ctx`, then assign to `self.engine` here.